packages/wrangler/src/config/environment.ts

```diff
@@ -1119,6 +1119,29 @@ export interface EnvironmentNonInheritable {
 		/** Whether the timer is enabled */
 		enable_timer?: boolean;
 	}[];
+
+	/**
+	 * Specifies rate limit bindings that are bound to this Worker environment.
+	 *
+	 * NOTE: This field is not automatically inherited from the top level environment,
+	 * and so must be specified in every named environment.
+	 *
+	 * @default []
+	 * @nonInheritable
+	 */
+	ratelimit: {
+		/** The binding name used to refer to the rate limiter in the Worker. */
+		name: string;
```

Comment:
I won't block on this because we're not 100% consistent, but I'd lean towards this being called `binding` rather than name